### PR TITLE
import retirements view and components to carbonmark

### LINKED ISSUE #1095: [retirements] Import /retirements pages to Carbonmark as-is
This ticket replaces #1018 which originally described more comprehensive re-write of how we fetch data and render the pages and handle 404s.

Instead, in the interest of time, I think we should do a hard copy-paste of our existing /retirements/[address] and /retirements/[address]/[index] pages to carbonmark, tweaking only the design items highlighted in #1088 

After this ticket is complete, @theprofessora should have by then completed the fully redesigned carbonmark version of the retirement pages. When we implement the new designs would be a good time to consider the enhancements described in #1096 and #1018

# Considerations

Some of the elements in the KlimaDAO retirements pages are not relevant in carbonmark. We have highlighted these in #1088

### PR DESCRIPTION
**Pull request by 0xMakka**
## Description

Copy retirements view into Carbonmark as-is apart from some style changes and removal of sections according to the details specified in #1095.

Note: The example in the screenshots below is using the following beneficiary address - 0xa83148b098038f366534253704ae4ff3cf931b94

## Related Ticket

Resolves #1095 

## Changes

screencapture-localhost-3002-retirements-0xa83148b098038f366534253704ae4ff3cf931b94-2023-04-29-13_09_54

screencapture-localhost-3002-retirements-0xa83148b098038f366534253704ae4ff3cf931b94-15-2023-04-29-13_10_32

## Checklist

- [X] I have run `npm run build-all` without errors
- [X] I have formatted JS and TS files with `npm run format-all`

### CODE REVIEW

**Comment by jabby09** [comment]
@0xMakka @Atmosfearful here is some better temporary copy for the About Carbonmark section.

> Carbonmark is an all-in-one platform for the carbon markets. The platform allows users to quickly list, acquire and offset Digital Carbon Credits with no additional fees. Carbonmark aims to reduce barriers to entry to the Digital Carbon Market, and empower market participants to better interact with one another. With millions of carbon credits available, get started on your journey today at carbonmark.com. Learn more on our [resources page](https://www.carbonmark.com/resources).

**Comment by jabby09** [comment]
@0xMakka I noticed I missed a KlimaDAO reference.

Under Project Details...

Change "Every KlimaDAO retirement..." to "Every Carbonmark retirement..."

**Comment by 0xMakka** [comment]
> @0xMakka I noticed I missed a KlimaDAO reference.
> 
> Under Project Details...
> 
> Change "Every KlimaDAO retirement..." to "Every Carbonmark retirement..."

Good catch, thanks @jabby09 will make this change also.

**Comment by jabby09** [comment]
@0xMakka not sure this is the correct ticket/place for this but also need to review all the social share button URL paths and make sure they are working with Carbonmark. For example twitter...change @klimadao to @carbonmarkcom

Screenshot 2023-04-29 at 8 47 50 AM

**Comment by 0xMakka** [comment]
> @0xMakka not sure this is the correct ticket/place for this but also need to review all the social share button URL paths and make sure they are working with Carbonmark. For example twitter...change @klimadao to @carbonmarkcom
> 
> Screenshot 2023-04-29 at 8 47 50 AM

Yeah I'll make this change as well in this PR. I see where I missed the twitter handle on this, so will make sure they are all changed to carbonmark 👌

**Comment by jabby09** [comment]
@0xMakka looking good!

I don't really use facebook so hard to test that one but might still need a small tweak for LI

Screenshot 2023-04-29 at 10 13 50 AM

File: carbonmark/lib/api.ts [hunk 1]
```diff
@@ -167,6 +169,23 @@ export const getVintages = async (): Promise<string[]> => {
   return data;
 };
 
+export const getDownloadCertificate = async (
```
Comment (0xMakka) [inline carbonmark/lib/api.ts:172]: @Atmosfearful updated to use the new api for generating the certificate and removed the other unused files for generating certificates in /carbonmark. Let me know if this looks okay.

File: carbonmark/lib/api.ts [hunk 2]
```diff
@@ -167,6 +169,23 @@ export const getVintages = async (): Promise<string[]> => {
   return data;
 };
 
+export const getRetirementCertificate = async (
```
Comment (0xMakka) [inline carbonmark/lib/api.ts:172]: @Atmosfearful updated to use the new api for generating the certificate and removed the other unused files for generating certificates in /carbonmark. Let me know if this looks okay.

**Review by Atmosfearful — APPROVED** [review]

**Review by Atmosfearful — APPROVED** [review]

**Pull request merged**